addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -320,14 +322,14 @@
                         <img class="img img-fluid position-absolute w-100 h-100 o_img_product_cover" loading="lazy" t-att-src="data['image_512']" t-att-alt="record.display_name"/>
                     </a>
                     <div class="o_carousel_product_card_body d-flex flex-column justify-content-between h-100 bg-black-50 p-3 position-relative">
-                        <div class="mb-3">
+                        <div class="mb-1">
```

Comment:
Decreasing this margin creates a spacing inconsistency when the rating stars are displayed.
I'm not sure if changing this class is necessary.

<img width="616" alt="Capture d’écran 2022-11-10 à 12 11 02" src="https://user-images.githubusercontent.com/80679690/201076268-b3273043-d4dc-47cf-a1a3-58e0b6f5a98c.png">